pkg/util/k8sutil/k8sutil.go

```diff
@@ -134,6 +136,21 @@ func ImageName(repo, version string) string {
 	return fmt.Sprintf("%s:v%v", repo, version)
 }
 
+func ImageNameBusybox(policy *api.PodPolicy) string {
```

Comment:
Make this method private by uncapitalize it imageName...
It is not used outside this package.